cid/helpers/quicksight/version.py

```diff
@@ -0,0 +1,63 @@
+import re
+import logging
+
+logger = logging.getLogger(__name__)
+
+class CidVersion:
+    
+    def __init__(self, version):
+        
+        if isinstance(version, __class__):
+            self.major, self.minor, self.build = version.major, version.minor, version.build 
+        elif isinstance(version, str):
+            self.major, self.minor, self.build = self._parse(version)
+        else:
+            raise TypeError(f'{version} must be {__class__} or str ')
+
+    def __str__(self):
+        return f'v{self.major}.{self.minor}.{self.build}'
+    
+    def _parse(self, str_version):
+        
+        version_pattern = re.compile(r"^[v|V](?P<major>[0-9]+)\.(?P<minor>[0-9]+)\.(?P<build>[0-9]+)$")
+        results = version_pattern.match(str_version)
+        
+        if not results:
+            logger.debug(f'Could not find version pattern in provided string: {str_version}')
+            major = 1
```

Comment:
This is an interim workaround to cater for all template based dashboards that may not follow the right convention for the dashboard version ( v<major>.<minor>.<build> ) and avoid them from breaking the cid-cmd execution.